source/about/self-hosted-subscriptions.rst

```diff
@@ -82,8 +83,8 @@ This process adds additional users to your existing subscription and is not a ne
 
 If you don't have a self-hosted license in the Customer Portal and want to add users, `contact us <https://mattermost.com/contact-us/>`__.
 
-Quarterly true-up report
-------------------------
+Quarterly true-up report prior to v7.9
+--------------------------------------
 
 When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as **Inactive** in **System Console > User Management > Users**.
```

Comment:
We don't refer to "registered users" anywhere else - is this meant to be "active users"? :)

Reply:
I think "active users" makes sense here. Just an FYI, I found a few other instances that could use a change (in another pr) if we want to avoid "registered users"

https://github.com/mattermost/docs/search?q=%22registered+user%22%2C%22registered+users%22

Reply:
Heads up I think the source of that "registered users" language comes from our legal terms: https://mattermost.com/software-services-license-agreement/ (Section 3.3b)

"Activated users" in my opinion is the one that best describes these users, but I believe since our terms still reflect "Registered Active Users", "active users" is the next best thing